src/NHibernate/Type/NullableType.cs

```diff
@@ -82,38 +82,40 @@ protected NullableType(SqlType sqlType)
 		/// </remarks>
 		public abstract object Get(DbDataReader rs, string name, ISessionImplementor session);
 
-
 		/// <summary>
 		/// A representation of the value to be embedded in an XML element 
 		/// </summary>
 		/// <param name="val">The object that contains the values.
 		/// </param>
 		/// <returns>An Xml formatted string.</returns>
-		public abstract string ToString(object val);
+		// Since 5.2
+		[Obsolete("This method has no more usages and will be removed in a future version. Override ToLoggableString instead.")]
+		public virtual string ToString(object val)
+		{
+			return val.ToString();
+		}
 
 		/// <inheritdoc />
-		/// <remarks>
-		/// <para>
-		/// This implementation forwards the call to <see cref="ToString(object)"/> if the parameter 
-		/// value is not null.
-		/// </para>
-		/// <para>
-		/// It has been "sealed" because the Types inheriting from <see cref="NullableType"/>
-		/// do not need and should not override this method.  All of their implementation
-		/// should be in <see cref="ToString(object)"/>.
```

Comment:
So a thing meant to log plain text is implemented by calling a thing documented as meant to generate xml strings...